app/controllers/admin_controller.rb

```diff
@@ -0,0 +1,25 @@
+class AdminController < ApplicationController
```

Comment:
It's a best practice to have the class naming agree with what actions are performed. Since these are actions of an admin on applicants, this class could be `Admin::ApplicantsController`.